src/AltinnCore/Common/Configuration/GeneralSettings.cs

```diff
@@ -10,6 +10,8 @@ public class GeneralSettings
         /// </summary>
```

Comment:
Looks like the is some indentation problem in this file, in Visual Studio go to "Edit" -> "Advanced" -> "Format Document" and se if it fixes the indentation differences in this file

Reply:
done